package.json

```diff
@@ -39,8 +39,10 @@
     "pull-stream": "^3.6.14",
     "push-stream": "^11.0.0",
     "rimraf": "^3.0.2",
+    "ssb-bendy-butt": "~0.12.0",
```

Comment:
Both `ssb-bendy-butt` and `ssb-uri2` have newer versions (`0.12.2` and `1.5.2` respectively). Maybe you'd like to update them here?